Traceback (most recent call last):
  File "c:\anaconda3\lib\site-packages\jupyter_cache\executors\utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "c:\anaconda3\lib\site-packages\nbclient\client.py", line 1087, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "c:\anaconda3\lib\site-packages\nbclient\util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "c:\anaconda3\lib\site-packages\nbclient\util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "c:\anaconda3\lib\asyncio\base_events.py", line 587, in run_until_complete
    return future.result()
  File "c:\anaconda3\lib\site-packages\nbclient\client.py", line 541, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "c:\anaconda3\lib\site-packages\nbclient\client.py", line 832, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "c:\anaconda3\lib\site-packages\nbclient\client.py", line 740, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply['content'])
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
num_puntos = 301
x_min = -15
x_max = -x_min
incremento = (x_max - x_min)/(num_puntos - 1) # Hay (num_puntos - 1) intervalos incluyendo x_min y x_max

lista_x = []
for i in range(num_puntos):
    x = x_min + i*incremento
    lista_x.append(x)

lista_y = [math.sin(x)/x for x in lista_x]
    
plt.plot(lista_x,lista_y)
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mNameError[0m                                 Traceback (most recent call last)
[1;32m<ipython-input-6-5f084ff8e500>[0m in [0;36m<module>[1;34m[0m
[0;32m      9[0m     [0mlista_x[0m[1;33m.[0m[0mappend[0m[1;33m([0m[0mx[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0;32m     10[0m [1;33m[0m[0m
[1;32m---> 11[1;33m [0mlista_y[0m [1;33m=[0m [1;33m[[0m[0mmath[0m[1;33m.[0m[0msin[0m[1;33m([0m[0mx[0m[1;33m)[0m[1;33m/[0m[0mx[0m [1;32mfor[0m [0mx[0m [1;32min[0m [0mlista_x[0m[1;33m][0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m     12[0m [1;33m[0m[0m
[0;32m     13[0m [0mplt[0m[1;33m.[0m[0mplot[0m[1;33m([0m[0mlista_x[0m[1;33m,[0m[0mlista_y[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m

[1;32m<ipython-input-6-5f084ff8e500>[0m in [0;36m<listcomp>[1;34m(.0)[0m
[0;32m      9[0m     [0mlista_x[0m[1;33m.[0m[0mappend[0m[1;33m([0m[0mx[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0;32m     10[0m [1;33m[0m[0m
[1;32m---> 11[1;33m [0mlista_y[0m [1;33m=[0m [1;33m[[0m[0mmath[0m[1;33m.[0m[0msin[0m[1;33m([0m[0mx[0m[1;33m)[0m[1;33m/[0m[0mx[0m [1;32mfor[0m [0mx[0m [1;32min[0m [0mlista_x[0m[1;33m][0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m     12[0m [1;33m[0m[0m
[0;32m     13[0m [0mplt[0m[1;33m.[0m[0mplot[0m[1;33m([0m[0mlista_x[0m[1;33m,[0m[0mlista_y[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m

[1;31mNameError[0m: name 'math' is not defined
NameError: name 'math' is not defined

